Sources/Networking/HTTPClient/SimpleNetworkService.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SimpleNetworkService.swift
+//
+//  Created by Jacob Zivan Rakidzich on 8/12/25.
+
+import Foundation
+
+/// A protocol representing a simple network service
+@_spi(Internal) public protocol SimpleNetworkService {
```

Comment:
Nit: I'd consider naming this `SimpleNetworkServiceType` to be consistent with how other protocols in the codebase are named